playbooks/prerun-network-checks.yml

```diff
@@ -1,188 +1,199 @@
 ---
+# This playbook is used to run preliminary network checks for the hosts in the ACS deployment.
+# the default('localhost') is only useful while testing on molecule
 - name: Run preliminary network checks for repository hosts
   hosts: repository
   become: true
-  roles:
-    - role: '../roles/helper_modules'
   tasks:
+    - name: Include common defaults
+      ansible.builtin.include_vars: ../roles/common/defaults/main.yml
+
     - name: Check db connection
-      ansible.builtin.include_tasks: "../roles/helper_modules/tasks/check_port.yml"
+      ansible.builtin.include_tasks: "tasks/check_port.yml"
       vars:
-        checked_host: "{% if groups.database | length == 0 %}127.0.0.1{% else %}{{ hostvars[groups.database[0]].ansible_host | default('127.0.0.1') }}{% endif %}"
+        checked_host: "{{ hostvars[groups.database[0]].ansible_host | default('localhost') }}"
```

Comment:
actually we should probablyuse `{{ db_host }}` as that's what's used in the repo role to connect to the db
https://github.com/Alfresco/alfresco-ansible-deployment/blob/fe23ffa25bea1ee545a69f57b8f555e88d2e03c4/roles/common/defaults/main.yml#L43